tests/relational/test_relational_data_with_json.py

```diff
@@ -713,6 +713,150 @@ def test_restore_with_empty_tables(bball, invented_tables):
     assert jimmy["suspensions"] == []
 
 
+@pytest.fixture
+def nested_listed_objects(tmpdir):
+    json = """
+{
+    "Records": [
+        {
+            "userAgent": "hello",
+            "responseElements": {
+                "accountAttributes": [
+                    {
+                        "attributeName": "duration",
+                        "attributeValues": [
+                            {
+                                "attributeValue": "45"
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+    ]
+}
+"""
+    json_df = pd.read_json(json)
+
+    rel_data = RelationalData(directory=tmpdir)
+    rel_data.add_table(name="demo", primary_key=None, data=json_df)
+
+    return rel_data
+
+
+def test_nested_listed_objects(nested_listed_objects):
```

Comment:
`
```suggestion
def test_nested_list_objects(nested_listed_objects):
```

Reply:
`list_objects` feels just slightly ambiguous to me, I'm going with `nested_lists_of_objects`